ui/uihandler.go

```diff
@@ -265,8 +275,53 @@ func Handler(w http.ResponseWriter, r *http.Request) {
 			}
 			uiRunMapMutex.Unlock()
 		}
-	case run:
-		// mode == run case:
+	case rungrpc:
```

Comment:
more code should be shareable between the 2 cases

see how fortio_main.go does grpc/http with sharing the interface between the 2